html_css_exercise/femc_level1_huddleLandingPage/css/index.css

```
@@ -0,0 +1,144 @@
+:root {
+    --background-color-1: #674bb1;
+    font-size: 62.5%;
+}
+
+html,
+body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    overflow: hidden;
+}
+
+body {
+    background-image: url(../img/bg-desktop.svg);
+    background-color: var(--background-color-1);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+a {
+    cursor: pointer;
+}
+
+/****** Header Styling *********/
+
+.header-main {
+    width: 100%;
+    height: 65px;
+    margin: 0 0 40px;
+    padding: 0 40px;
+}
+
+/****** Main Section Styling *********/
+
+.container-main {
+    width: 100%;
+    padding: 0 40px;
+    display: flex;
+}
+
+.main-img-mockup {
+    flex-grow: 1;
+    max-width: 1000px;
+    margin-bottom: 40px;
+}
+
+.main-img-mockup img {
+    width: 100%;
+    height: auto;
+}
+
+.main-message {
+    width: 40%;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #fff;
+    padding: 0 0 60px 40px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.main-message h2 {
+    font-size: 3.6rem;
+    font-weight: 600;
+    line-height: 1;
+    padding: 0 0 40px;
+}
+
+.main-message p {
+    font-size: 1.8rem;
+    line-height: 1.25;
+    padding: 0 0 40px;
+}
+
+.main-button-register a {
+    font-size: 1.8rem;
+    line-height: 1.25;
+    font-weight: 600;
+    text-transform: uppercase;
+    background-color: #fff;
+    color: var(--background-color-1);
+    padding: 5px 20px;
+    border-radius: 60px;
+    transition: all ease-in-out 100ms;
+}
```

Took a stab at trying to clean this up and get it closer to your desktop-design. 
- Updated padding and font size (assuming you remove the root font override)
- Remove unnecessary line-height, font-weight, text-transform
- Add subtle box-shadow

```suggestion
.main-button-register a {
    font-size: 1.4rem;
    background-color: #fff;
    box-shadow: 3px 3px 16px 0px #00000082;
    color: var(--background-color-1);
    padding: 1.5rem 5rem;
    border-radius: 60px;
    transition: all ease-in-out 100ms;
}
```

---

box-shadow is a great addition. I originally only added the hover state to make the button seem more "alive" the box-shadow accomplishes that well